Mathlib/Topology/Algebra/Module/FiniteDimension.lean

```diff
@@ -237,23 +238,25 @@ private theorem continuous_equivFun_basis_aux [T2Space E] {ι : Type v} [Fintype
     change Continuous (ξ.coord i)
     exact H₂ (ξ.coord i)
 
-/-- Any linear map on a finite dimensional space over a complete field is continuous. -/
-theorem LinearMap.continuous_of_finiteDimensional [T2Space E] [FiniteDimensional 𝕜 E]
-    (f : E →ₗ[𝕜] F') : Continuous f := by
+variable (E 𝕜) in
+theorem FiniteDimensional.isModuleTopology [T2Space E] [FiniteDimensional 𝕜 E] :
+    IsModuleTopology 𝕜 E :=
```

Comment:
Can we make this an instance?

Reply:
Apparently it breaks things with `IsModuleTopology.toContinuousSMul`, but maybe that shouldn't be an instance

Reply:
Can you please merger master and see if it still breaks something?

Reply:
it indeed still breaks without scoping or removing the `IsModuleTopology.toContinuousSMul` instance

Reply:
I don't understand how making this an instance could cause trouble. I'm going to merge master (even though this shouldn't affect anything) and investigate now.

Reply:
In the end, we did not make `isModuleTopologyOfFiniteDimensional` a global instance so we can resolve this.